templates/SilverStripe/MFA/Email/AccountReset.ss

```diff
@@ -0,0 +1,5 @@
+<p><%t SilverStripe\\Control\\ChangePasswordEmail_ss.HELLO 'Hi' %> $Member.FirstName,</p>
```

Comment:
I think you should include the name as part of the translation e.g. `"Hi {name}"` - not sure about the comma, perhaps also in the translation

Reply:
There's no variation of the core translation that includes the name by the looks of things - I'm inclined to leave this as-is for maximum localisation.